lib/notifications/notification_server.ex

```diff
@@ -108,7 +119,7 @@ defmodule Notifications.NotificationServer do
           nil -> {nil, nil, nil}
         end
 
-      %Notification{
+      %{
```

Comment:
Any particular reason for turning this from a struct to a plain map?

Reply:
Functionally it would work fine as a struct, but semantically it's not a notification itself--it's a collection of values that we are proposing to construct a notification from.